### ISSUE TITLE
Incorrect production numbers

### ISSUE THREAD

**Issue by ACPixel** [porisius/FicsitRemoteMonitoring#16]
Hey there! I'm currently really enjoying this mod, and am working on building out a new web dashboard for it. However while using the getProdStats endpoint, a lot of the numbers seem to be wrong. I'm not sure if I'm missing something but. The consuption numbers *seem* to be correct, however the actual production numbers, even for things I am producing hundreds or thousands of per minute, seem to all say small numbers like 0-2.

Maybe I'm just missing something, but figured I'd open an issue to see!

**Comment by ACPixel** [comment]
Looking more carefully, it looks like the API might actually be giving the production numbers per second and not per minute, multiplying by 60 looks to get roughly the correct numbers.

**Comment by porisius** [comment]
I think I might have found the error, but do you have the save you were using or I can send you a dev copy with that fix in place? Your call.

**Comment by ACPixel** [comment]
Happy to upload the save or download a test version, either way! Going to be out on vacation in a couple days though so not sure when I can do either of those. Let me know which is easier for you!

**Comment by porisius** [comment]
[FicsitRemoteMonitoring.zip](https://github.com/porisius/FicsitRemoteMonitoring/files/8427575/FicsitRemoteMonitoring.zip)
Here you go, I had been busy with life... so when you get a chance. I might have to relook it over again, but I think I got it.

**Comment by captncraig** [comment]
I'm also seeing odd numbers in the latest version. It looks to me to be a factor of 75 though, not 60. In my very simple setup I am producing `45` limestone per minute. `/stats` in the stats mod confirms this. The `/getProdStats` endpoint though, only shows `.6`. `45/.6 = 75`. I haven't looked into the code to see why, but the patched version above does not seem to fix. It actually makes me see zeros for some production where I know I see things going.

**Comment by captncraig** [comment]
Oddly, the consumption numbers are correct, but the production are off.

**Comment by GalaxyWolfYT-Official** [comment]
> 

Might be desync

**Comment by porisius** [comment]
Nah, it is a calculation issue... luckily/unfortunately, I borked up the BP function that was running it trying to revert to a backup, and I lost about 20% of the nodes (which is still a lot). But, it allowed me to clean things up and fix this particular problem.

I'll be releasing a new version soon.

**Comment by porisius** [comment]
Version is updated, when you get a chance, please take a look and see if it corrects your issue. I didn't see any issues, but I also did a miner and a constructor making concrete.

**Comment by ACPixel** [comment]
From a quick look at things it does seem to be fixed! 

Thank for for getting that done :D
